CHANGELOG.md

```diff
@@ -13,6 +13,27 @@ All notable changes to this project will be documented in this file.
 This project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.38.0]
```

Comment:
0.37.2 ? given that I've fixed the issue that drove the unexpose of pseudonymisation API